RISTNet.h

```diff
@@ -239,6 +239,9 @@ class RISTNetReceiver {
   std::function<std::shared_ptr<NetworkConnection>(std::string lIPAddress, uint16_t lPort)>
       validateConnectionCallback = nullptr;
 
+  /// Callback handling disconnecting clients
+  std::function<void()> clientDisconnectedCallback = nullptr;
```

Comment:
I haven't looked into the RIST code much, but it seems like it is possible to have multiple clients, right? (mClientList)
In that case the clientDisconnectCallback should also tell which client disconnected.

Reply:
Good point, I'll look into that!

Reply:
Ok, rist_peer is now in the callback. I wonder, maybe also the NetworkConnection object also?

Reply:
Yes, I think we should include it also. It can be used for carrying user defined data which might be good to get hold of in case of a disconnection.